### DISCUSSION TITLE
Extract multi-part archive files

### DISCUSSION THREAD

**Discussion by vlaved** [aonez/Keka#1078] [Q&A] [answered]
Hello,

How can I extract google multi-part archive files like this? Couldn't find it in settings.
![google](https://user-images.githubusercontent.com/30136182/160290794-93bf65f2-c298-41f0-870d-d1e60cb5925b.png)

**Comment by aonez** [comment, marked as answer]
Just drop the first to Keka 😊

**Reply by vlaved** [reply]
Thx! It seems like Google photo archive has his own features. It's not a multi-part archive actually...

**Comment by Jcarrera92** [comment]
I still can't extract multi-part files, it is still extracting all of them in separate folders.

**Reply by aonez** [reply]
Can you share an example file that replicate your issue?

**Comment by ivanglushko** [comment]
Encountered the same issue. Keka can't recognize a multipart zip. If i drop the first one it just extracts it. And if i select all of them it extracts all of them as well. 
And i expect as a result to have 1 folder with merged contents

**Reply by ivanglushko** [reply]
What worked for me (Mac):
- `$ mkdir merged` // make a folder for extracted zip
- `$ unzip 'assembly*.zip' -d merged` // unzip all zips matching name and wildcard to specified directory

**Reply by Develeone** [reply]
Ещё вариант под Mac, с поддержкой кириллицы:

`find . -name "Архив-*.zip" -exec ditto -xk {} merged \;`
